tutorial_code/http-file-server/file-server.toit

```diff
@@ -0,0 +1,122 @@
+import host.file
```

Comment:
Maybe call this `file_server.toit` for now?

Reply:
done.